packages/react-components/src/components/Modal/components/ActionModalContent.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+
+import cx from 'clsx';
+
+import { Heading, Text } from '../../Typography';
+
+import styles from './ActionModalContent.module.scss';
+
+const baseClass = 'action-modal-content';
+
+interface IActionModalProps {
+  /**
+   * Optional element to render icon or image
+   */
+  icon?: React.ReactNode;
+  /**
+   * Set the header text
+   */
+  heading?: React.ReactNode;
+  /**
+   * Optional element to render action buttons
+   */
+  actions?: React.ReactNode;
+  /**
+   * The CSS class for main container
+   */
+  className?: string;
+  /**
+   * The CSS class for header container
+   */
+  headerClassName?: string;
+  /**
+   * The CSS class for content container
+   */
+  contentClassName?: string;
+}
+
+export const ActionModalContent: React.FC<IActionModalProps> = ({
```

Comment:
Is it a new component for external usage? We need to document it in our docs: what its purpose, when and how it should be used.

Reply:
It's a sub-component used inside `Modal` (like `Info` or `Interactive` sub-components in `Tooltip`).

Reply:
(replied on priv)
TLDR: we need a docs describing sub-components and their use-cases.